cli/pkg/localcatalog/tracking_plan.go

```diff
@@ -7,9 +7,10 @@ import (
 )
 
 var (
-	PropRegex    = regexp.MustCompile(`^#\/properties\/(.*)\/(.*)$`)
-	EventRegex   = regexp.MustCompile(`^#\/events\/(.*)\/(.*)$`)
-	IncludeRegex = regexp.MustCompile(`^#\/tp\/(.*)\/event_rule\/(.*)$`)
+	PropRegex       = regexp.MustCompile(`^#\/properties\/(.*)\/(.*)$`)
+	EventRegex      = regexp.MustCompile(`^#\/events\/(.*)\/(.*)$`)
+	IncludeRegex    = regexp.MustCompile(`^#\/tp\/(.*)\/event_rule\/(.*)$`)
+	CustomTypeRegex = regexp.MustCompile(`^#\/custom-types\/(.*)\/(.*)$`)
```

Comment:
name should start with capital letter

Reply:
You mean the custom type's name as created in the backend? Is this a constraint we enforce already in our APIs?
For reference this regex is for references in YAML files, which require the custom type's local id, not the name. So this type of validation should be implemented in a different place, where we validate the name property in the custom type spec.